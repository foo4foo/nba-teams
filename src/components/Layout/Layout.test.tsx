import React from 'react'
import { BrowserRouter as Router } from 'react-router-dom'
import renderer from 'react-test-renderer'
import { Layout } from './'

describe('DataTable', () => {
  it('renders successfuly', () => {
    const tree = renderer
      .create(
        <Router>
          <Layout renderRoutes={() => <React.Fragment />} />
        </Router>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
