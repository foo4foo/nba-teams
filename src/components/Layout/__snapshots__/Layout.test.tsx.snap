// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataTable renders successfuly 1`] = `
<div
  className="StyledGrid-sc-1wofa1l-0 dgUwaQ"
>
  <div
    className="StyledBox-sc-13pk1d4-0 brLKxe"
  >
    <button
      className="StyledButton-sc-323bzc-0 jfbKsv"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onMouseOut={[Function]}
      onMouseOver={[Function]}
      type="button"
    >
      <span
        className="StyledText-sc-1sadyjn-0 jScuyG"
        size="large"
      >
        NBA
      </span>
    </button>
  </div>
  <div
    className="StyledBox-sc-13pk1d4-0 mNTkb"
  >
    <a
      aria-current="page"
      className="active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      <button
        className="StyledButton-sc-323bzc-0 jfbKsv"
        onBlur={[Function]}
        onFocus={[Function]}
        onMouseOut={[Function]}
        onMouseOver={[Function]}
        type="button"
      >
        <div
          className="StyledBox-sc-13pk1d4-0 hcMawI"
        >
          <span
            className="StyledText-sc-1sadyjn-0 eTqRlr"
          >
            Home
          </span>
        </div>
      </button>
    </a>
    <a
      aria-current={null}
      href="/favorites"
      onClick={[Function]}
    >
      <button
        className="StyledButton-sc-323bzc-0 jfbKsv"
        onBlur={[Function]}
        onFocus={[Function]}
        onMouseOut={[Function]}
        onMouseOver={[Function]}
        type="button"
      >
        <div
          className="StyledBox-sc-13pk1d4-0 hcMawI"
        >
          <span
            className="StyledText-sc-1sadyjn-0 eTqRlr"
          >
            Favorites
          </span>
        </div>
      </button>
    </a>
  </div>
  <div
    className="StyledBox-sc-13pk1d4-0 lbcOS"
  />
</div>
`;
